picard/file.py

```diff
@@ -258,7 +260,7 @@ def _preserve_times(self, filename, func):
             # https://docs.python.org/3/library/os.html#os.utime
             # Since Python 3.3, ns parameter is available
             # The best way to preserve exact times is to use the st_atime_ns and st_mtime_ns
-            # fields from the os.stat() result object with the ns parameter to utime.
+            # fields from the os.stat() result object with the ns parameter to utime.
```

Comment:
If someone else is wondering: This is changing from a non-breaking space to a regular space. It's strictly speaking an unrelated change, but I don't mind including this minor cleanup in a commit like this. But if someone has a strong opinion on this we could move it to a separate commit.